tests/contribution_bounders_test.py

```diff
@@ -150,3 +152,58 @@ def test_contribution_bounding_empty_col(self):
         bound_result = self._run_contribution_bounding(input, max_contributions)
 
         self.assertEmpty(bound_result)
+
+
+class LinfSampler(parameterized.TestCase):
```

Comment:
nit: I would vote for adding blank lines between arrange, and and assert sections to make it clear what's being tested.

Reply:
Done